dbt/contracts/graph/parsed.py

```diff
@@ -252,6 +252,9 @@
     'properties': {
         'nodes': PARSED_NODES_CONTRACT,
         'macros': PARSED_MACROS_CONTRACT,
+        'generated_at': {
+            'type': 'date-time'
```

Comment:
the correct json schema draft4 type for this is:

```json
{
    "type": "string",
    "format": "date-time"
}
```

Reply:
So, I was wondering how tests were passing after this comment - it turns out we never use this schema at all. Should I just remove it entirely?

Reply:
we need at least one json schema for the manifest, and at least one for the catalog. it's actually really important as a piece of documentation to share with the community. so ideally we'd either (a) apply it or (b) just leave it in here but do our best to correct it, even though it's not currently used. up to you which you want to do here

Reply:
I've changed the ParsedManifest to validate the schema (and made sure it fails on the old schema).